package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tailwindcss-capsize",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
No need to increase this manually, it'll be bumped automatically during the release workflow 👍